src/calculateRentalCost.js

```diff
@@ -28,7 +28,24 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (typeof days !== 'number' || !Number.isInteger(days) || days < 1) {
+    throw new Error('The input parameter must be a positive integer');
+  }
+
+  const dailyCost = 40;
```

Comment:
Consider using a constant for the minimum number of days for the basic discount (3) and the extra discount (7).